CSS Folder/IndexStyle.css

```diff
@@ -0,0 +1,193 @@
+@font-face {
+    font-family: "Roboto";
+    src: url("./Fonts/Roboto/Roboto-Regular.ttf") format("truetype");
+}
+
+* {
+    font-family: "Roboto", sans-serif;
+}
+
+body {
+    background: radial-gradient(circle, #b597f6 30%, #96c6ea 100%);
+}
+
+main {
+    margin: 0 auto;
+    width: 90%;
+}
+
+h1 {
+    font-size: 42px;
+    text-align: center;
+    text-decoration: none;
+    margin-top: 10px;
+    margin-bottom: 20px;
+}
+
+h2 {
+    font-size: 28px;
+    color: black;
+    text-align: center;
+}
+
+h2.importantsubtitle {
```

Comment:
remember to always do the more general selectors on top of your files and keep the more specific at the bottom, so css knows what's more important/specific